src/components/ec-option-card/ec-option-card.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <component
+    :is="componentTag"
+    v-bind="{
+      ...attrs,
+      ...componentProps,
+      'data-test': attrs['data-test'] ? `${attrs['data-test']} ec-option-card` : 'ec-option-card',
+    }"
+    class="ec-option-card"
+    :class="getOptionCardClasses()"
+    @click="setClickAction()"
+  >
+    <span
+      data-test="ec-option-card__title"
+    >
+      <ec-icon
+        v-if="optionCardIconName && optionTitle"
+        data-test="ec-option-card__title-img"
+        class="ec-option-card__title-img"
+        :class="getOptionIconClass()"
+        :name="optionCardIconName"
+        :alt="optionTitle"
+        :size="15"
```

Comment:
Should it be 16?

Reply:
Why 16?
I put 15 because the design had 14.4px so I went for 14 because 13 was too small. 
![image](https://user-images.githubusercontent.com/15978652/220639721-44ac91c9-954c-472e-8f9d-e18d8399de60.png)

Reply:
I see 15 now, but we have no such size of icon for now. Maybe we could clarify this with Alba?

Reply:
Spoke with Alba - should be actually 24px... so will set it to that and fix the alignments. Apparently we've been exporting all the icons wrong from Figma... Alba will explain more...